.github/workflows/build.yaml

```diff
@@ -0,0 +1,53 @@
+name: Build
+on:
+  - push
+  - pull_request
+jobs:
+  build:
+    name: Build
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, macos-11]
+        qt: [5.15.2]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: recursive
+
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: ${{ matrix.os }}-${{ runner.arch }}
+          max-size: 100M
+
+      - name: Cache Qt
+        id: cache-qt
+        uses: actions/cache@v1  # not v2! TODO: why though?
```

Comment:
>

Reply:
I used the example from: [install-qt-action](https://github.com/jurplel/install-qt-action#cached). It isn't explained why not v2